src/services/LogFileManager/index.ts

```diff
@@ -301,7 +306,7 @@ class LogFileManager {
             "" :
             "i";
         const queryRegex = new RegExp(regexPattern, regexFlags);
-
+        this.#onQueryResults(0, new Map());
```

Comment:
```suggestion
        // Send an empty query result with 0 progress to the render to init the results variable because there could be results sent by the last task before `startQuery()` runs.
        this.#onQueryResults(0, new Map());
```